joinfaces-autoconfigure/pom.xml

```diff
@@ -221,6 +221,21 @@
             <artifactId>richfaces</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish.jaxb</groupId>
+            <artifactId>jaxb-runtime</artifactId>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jaxb</groupId>
```

Comment:
Isn't `javax.xml.bind` the correct groupId here?

see https://stackoverflow.com/a/43574427/3574494 and https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-with-Java-9#jaxb

Reply:
`org.glassfish.jaxb` is the jaxb reference implementation and jars are more updated.

Internet is funny: [Which artifacts should I use for JAXB RI in my Maven project?
](https://stackoverflow.com/questions/26413431/which-artifacts-should-i-use-for-jaxb-ri-in-my-maven-project)

Anyway, these libs are only used by richfaces. And this component library is retired.